anchor-service/src/merkle_tree.rs

```diff
@@ -0,0 +1,69 @@
+use anyhow::{anyhow, Result};
+use cid::Cid;
+
+use ceramic_core::DagCborIpfsBlock;
+use ceramic_event::anchor::{AnchorRequest, MerkleNode, MerkleNodes};
+
+pub struct MerkleTree {
+    pub root_cid: Cid,
+    pub nodes: MerkleNodes,
+    pub count: u64,
+}
+
+pub fn build_merkle_tree(anchor_requests: &[AnchorRequest]) -> Result<MerkleTree> {
+    if anchor_requests.is_empty() {
+        return Err(anyhow!("no requests to anchor"));
+    }
+    let mut peaks: [Option<Cid>; 64] = [None; 64];
```

Comment:
what are "peaks" here?  TBH I feel like this whole function could use a lot more comments explaining what it's doing

Reply:
Yes, will add more comments soon!

Reply:
Added comments from @AaronGoldman that we'd put in a later PR.